testing/package.json

```diff
@@ -9,7 +9,9 @@
     },
     "devDependencies": {},
     "scripts": {
-        "test": "npx backstop test"
+        "test": "npx backstop test",
```

Comment:
This stuff already in the parent package.json, no big harm putting it here but two things to keep in sync.

Reply:
I will remove to avoid confusion.